projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -203,4 +209,5 @@ export const defaultFeatureToggles: Required<FeatureTogglesInterface> = {
   a11yUnitsListKeyboardControls: false,
   a11yCartItemsLinksStyles: false,
   a11yFocusableCarouselControls: false,
+  cartQuickOrderRemoveListenToFailEvent: false,
```

Comment:
`RemoveListen` sounds quite funky due to 2 verbs in a row. What about `RemoveListening`?

PS. If we go for it, then let's also update the descriptions of unit tests mentioning the old toggle's name